hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -81,6 +70,18 @@
 import scala.Option;
 import scala.Tuple2;
 
+import java.io.IOException;
```

Comment:
In general, can you please avoid extra formatting changes with code reviews, it's hard to look at the file list and get a pulse on the PR.